### ISSUE TITLE
 removing the stray latin accesskey letters from bn-IN

### ISSUE THREAD

**Issue by birajkarmakar** [translate/pootle#4083]
Actually, we want to remove stray latin accesskey letters from bn-IN to make Firefox 42 more awesome. Please help me to get done. 

The problem is how can I find stray latin accesskey letters. Is there any way to find it easily.

**Comment by dwaynebailey** [comment]
@birajkarmakar I would use this search http://mozilla.locamotion.org/bn_IN/firefox/translate/#search=%26&sfields=target and then actually scan through the whole page and click on the units that are problematic to correct them.  This search will pick up variables like ``&browserShortName;`` so just scan the search results and select those that are an issue, that way you won't need to visit items that are not a problem.  I'll leave this issue open so you can ask for clarity and for you to close it when you are done.

**Comment by birajkarmakar** [comment]
Yes, but what I have seen here 1000+ strings are coming. But it would be good if we search like for this "&A" http://mozilla.locamotion.org/bn_IN/firefox/translate/#search=%26A&sfields=target&unit=6136963 . It help me to figure put whole thing if I get to you the whole access key list. Then I will find in that way.

@dwaynebailey what do you think?

**Comment by dwaynebailey** [comment]
@birajkarmakar not sure I understand what you want from me.  But going through each &[a-z] might be an easy way to do this.  The alternative is to work offline and use ``pogrep`` and ``pomerge``.  If you can craft a regex to find accesskeys and not get confused with &var; then that could also work.

**Comment by birajkarmakar** [comment]
@dwaynebailey I have removed all accesskey letters from bn-IN. Could you please cross check it anything is left. if everythin was ok please sign-off it .
